daemon/containerd/service.go

```
@@ -182,8 +182,10 @@ func (i *ImageService) GetContainerLayerSize(ctx context.Context, containerID st
 	}
 
 	c, err := i.client.ContainerService().Get(ctx, containerID)
+	// Get() returns an error if the container is not found, we return 0, 0 in this case and
+	// not an error because this will happen when the container exited.
 	if err != nil {
```

Do we need to check if it's a `errdefs.IsNotFound(err)`, to prevent we're not ignoring other errors ? https://github.com/containerd/containerd/blob/bbe46b8c43fc2febe316775bc2d4b9d697bbf05c/errdefs/errors.go#L56-L59

---

I guess we could ignore the error; perhaps for errors that are not "not found", we could log them; not sure if either is best handled in this function or by the caller though (haven't looked closely where it's all used)

---

Did you use the containerd errdefs? (not by accident the moby errdefs?)

---

OK nevermind, I imported the wrong package...

---

Fixed, thanks!

---

> 

Exactly what happened :D

---

> Did you use the containerd errdefs? (not by accident the moby errdefs?)

This is exactly what happened :D